api/audit/models.py

```diff
@@ -60,6 +61,13 @@ class AuditLog(models.Model):
         blank=True,
         on_delete=models.SET_NULL,
     )
+    master_api_key = models.ForeignKey(
```

Comment:
This is going to need to be discussed with @kyle-ssg I think. I know that AuditLog entries missing user objects have caused an issue in the past.

Reply:
So, with this change, we'll start having audit log entries with no author? That's fine if so, I can just list these as "System"

Reply:
Correct, yes.

Reply:
Resolving this as it looks like you've got a PR outstanding to handle it on the FE here: #1216.